charter.md

```diff
@@ -1,134 +1,238 @@
 # Charter of Data Distribution Working Group (DDWG)
 
+This charter outlines the necessary information that the DDWG requires to operate effectively.
+
+## Updates
+| Date | Changes |
+| --- | --- |
+| November 2013 | Initial Charter creation |
+| October 2018 | Hard drives no longer distributed regularly, BAS now distribution institution | 
+| May 2020 | Members list updated, mailing list change, definitions & policies added |
+
+## Definitions
+ - IQDAT/BFIQ: Beamformed IQ data
+ - RAWACF/DAT: Correlation data
+ - Level 0 Data: IQDAT or BFIQ data
+ - Level 1 Data: RAWACF or DAT data
+ - WG: Working group
+ - PI Institutions: Institutions that produce SuperDARN radar data adhering to the PI agreement
+ - DSWG: Data Standards Working Group: <https://github.com/SuperDARN/dswg-published-docs>
+ - DAWG: Data Analysis Working Group: TODO: link to some kind of page/charter/etc
+ 
+## Institutions
+ - BAS: British Antarctic Survey
+ - USASK: University of Saskatchewan
+ - VT : Virginia Tech
+ - NIPR: National Institute of Polar Research
+ - JHU/APL: Johns Hopkins University Applied Physics Laboratory
+ - UAF: University of Alaska Fairbanks
+ - IAPS/INAF: Institute for Space Astrophysics and Planetology 
+ - BOM: Bureau of Meteorology
+ - PRIC: Polar Research Institute of China
+ - ISEE: Institute for Space-Earth Environmental Research
+ - IRAP: Research Institute in Astrophysics and Planetology 
+ - NICT: National Institute of Information and Communications Technology
+ - DICE/UEC: University of Electro-Communications
```

Comment:
I could be mistaken, but "DICE" seems to be a (former?) departmental acronym while UEC refers to the institution.

Reply:
You're probably right, I couldn't find any reference to DICE, so I will remove it and only use UEC.